tests/shopify_python/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
+from shopify_python import google_styleguide
+from shopify_python import shopify_styleguide
```

Comment:
This is sortof hacky... Ideally we shouldn't need out tests to be contained within a package.